src/pbft/pbft.h

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/logger.h"
+#include "kv/kvtypes.h"
+#include "libbyz/libbyz.h"
+#include "libbyz/network.h"
+#include "libbyz/receive_message_base.h"
+#include "node/nodetypes.h"
+#include "pbft/pbft_deps.h"
+
+#include <list>
+#include <memory>
+#include <vector>
+
+namespace pbft
```

Comment:
Should we split this file into two: one for `PbftEnclaveNetwork` and one for `Pbft`.

Reply:
it was originally in pbft_deps.h. Don't really have a preference on this since I am guessing a lot of things will change and be re-shuffled in the future